addons/website_event_track/views/event_track_views.xml

```diff
@@ -100,10 +100,13 @@
             <search string="Event Tracks">
```

Comment:
Commit message:

- ease search tracks -> ease searching tracks
- Rename string of 'partner_id' field.(Contact => Speaker) -> That's actually misleading as you're not renaming any fields, just adding a string on a filter option, would not specify it (or not like that)